airbyte-protocol/models/src/main/resources/airbyte_protocol/airbyte_protocol.yaml

```diff
@@ -151,6 +151,9 @@ definitions:
           type: array
           items:
             type: string
+      namespace:
+        description: Optional Source-defined namespace. Currently only used by JDBC destinations to determine what schema to write to. Airbyte streams from the same sources should have the same namespace.
```

Comment:
Based on this description it feels like this concept should not live in the stream. If this information only makes sense to destinations then shouldn't it be defined in the connection or destination config?

Reply:
nvm this makes sense. We're just propagating the schema. I think I got a little thrown off by the description into thinking this is a destination feature that lives in the source.